modules/ROOT/pages/mule-high-availability-ha-clusters.adoc

```diff
@@ -4,6 +4,9 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: esb, deploy, configuration, cluster, ha, high availability, downtime
 
+[IMPORTANT]
+Clustering is not available in xref:runtime-manager::cloudhub.adoc[CloudHub], see xref:runtime-manager::cloudhub-fabric.adoc[CloudHub Fabric] for details of how workers can be shared or doubled to scale your application and provide high availability.
+
 A *cluster* is a set of Mule runtimes that acts as a unit. In other words, a cluster is a virtual server composed of multiple nodes. The nodes (Mule runtimes) in a cluster communicate and share information through a distributed shared memory grid. This means that the data is replicated across memory in different physical machines.
```

Comment:
Please go ahead and remove bold font from anything that's not a GUI label. Or Mysti will have to do it before we get it translated.